persistent/ChangeLog.md

```diff
@@ -2,6 +2,8 @@
 
 ## 2.11.0.1
 
+* [#1162](https://github.com/yesodweb/persistent/pull/1162)
+  * Replace `askLogFunc` with `askLoggerIO`
```

Comment:
2.11.0.1 has already been released on Hackage - since this is a breaking change, it'd need to go under 2.12 or 3.0.